crates/sui-core/src/signature_mpc/sign_round.rs

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) dWallet Labs, Ltd.
+// SPDX-License-Identifier: BSD-3-Clause-Clear
+
+use std::collections::HashSet;
+use std::mem;
+
+use futures::StreamExt;
+use rand::rngs::OsRng;
+
+use signature_mpc::twopc_mpc_protocols::decrypt::{
+    decrypt_signature_decentralized_party_sign, PartialDecryptionProof,
+};
+use signature_mpc::twopc_mpc_protocols::{
+    AdditivelyHomomorphicDecryptionKeyShare, DecentralizedPartyPresign, DecryptionPublicParameters,
+    DKGDecentralizedPartyOutput, Hash
+    , initiate_decentralized_party_sign, message_digest, PaillierModulusSizedNumber, PartyID
+    , ProtocolContext, PublicNonceEncryptedPartialSignatureAndProof, Result,
+    SecretKeyShareSizedNumber, SignatureThresholdDecryptionParty,
+};
+
+use crate::signature_mpc::identifiable_abort;
+use crate::signature_mpc::sign_state::SignState;
+
+#[derive(Default)]
+pub(crate) enum SignRound {
+    FirstRound {
+        signature_threshold_decryption_round_parties: Vec<SignatureThresholdDecryptionParty>,
+    },
+    #[default]
+    None,
```

Comment:
What is the purpose of an enum with one case?

Reply:
This code was originally in the `sign-ia-wasm` we only moved it to a different file. We decided not to refactor the original code, this will be done in a separate pr